### PR TITLE
[release/5.0] Add Fedora 35 RID

### PR DESCRIPTION
**Pull request by omajid**
Fedora rawhide now uses the fedora.35-x64 RID:

    $ podman run -it registry.fedoraproject.org/fedora:rawhide /bin/bash -c 'cat /etc/os-release'
    NAME=Fedora
    VERSION="35 (Container Image Prerelease)"
    ID=fedora
    VERSION_ID=35
    VERSION_CODENAME=""
    PLATFORM_ID="platform:f35"
    PRETTY_NAME="Fedora 35 (Container Image Prerelease)"
    ANSI_COLOR="0;38;2;60;110;180"
    LOGO=fedora-logo-icon
    CPE_NAME="cpe:/o:fedoraproject:fedora:35"
    HOME_URL="https://fedoraproject.org/"
    DOCUMENTATION_URL="https://docs.fedoraproject.org/en-US/fedora/rawhide/system-administrators-guide/"
    SUPPORT_URL="https://fedoraproject.org/wiki/Communicating_and_getting_help"
    BUG_REPORT_URL="https://bugzilla.redhat.com/"
    REDHAT_BUGZILLA_PRODUCT="Fedora"
    REDHAT_BUGZILLA_PRODUCT_VERSION=rawhide
    REDHAT_SUPPORT_PRODUCT="Fedora"
    REDHAT_SUPPORT_PRODUCT_VERSION=rawhide
    PRIVACY_POLICY_URL="https://fedoraproject.org/wiki/Legal:PrivacyPolicy"
    VARIANT="Container Image"
    VARIANT_ID=container

This is a backport of https://github.com/dotnet/runtime/pull/48200

### CODE REVIEW

**Comment by safern** [comment]
We need to bump Platforms package version and build it, right?

cc: @Anipik

**Comment by omajid** [comment]
> We need to bump Platforms package version and build it, right?

Done now.

**Comment by Anipik** [comment]
@rbhanda do we need this change for 5.0.4 ?

**Comment by Anipik** [comment]
> We need to bump Platforms package version and build it, right?

**Review by Anipik — CHANGES_REQUESTED** [review]
@omajid can you update the packageIndex.json as well ? https://github.com/dotnet/runtime/blob/release/5.0/docs/project/library-servicing.md

**Review by ViktorHofer — APPROVED** [review]

**Comment by omajid** [comment]
> can you update the packageIndex.json as well

Sure, running a `./build.sh` did the trick. I have no idea if all the changes are correct and desired, though.

**Review by ViktorHofer — COMMENTED** [review]
@Anipik can you please take a look at the other packageIndex.json changes in the PR?

**Review by Anipik — APPROVED** [review]

File: src/libraries/pkg/baseline/packageIndex.json [hunk 1]
```diff
@@ -1551,6 +1545,13 @@
       "BaselineVersion": "5.0.1",
       "InboxOn": {}
     },
+    "runtime.osx-x64.runtime.native.System.IO.Ports": {
```
Comment (safern) [inline src/libraries/pkg/baseline/packageIndex.json:1548]: @Anipik are these changes necessary or should we just update it for the platforms package?

Comment (Anipik) [reply src/libraries/pkg/baseline/packageIndex.json:1548]: its fine we can keep these, it will help to reduce the noise in future.

**Review by Anipik — APPROVED** [review]

**Comment by omajid** [comment]
Is this okay to merge now?

**Comment by omajid** [comment]
Thanks for merging this!
